dask/dataframe/core.py

```diff
@@ -4949,16 +4949,21 @@ def quantile(df, q, method="default"):
     else:
         internal_method = method
 
+    import sys
+
+    package_name = df.__class__.__module__.split(".")[0]
+    DD = sys.modules[package_name]
```

Comment:
Had to fetch package as I didn't find a way to construct a `Series` object with a `DataFrame` in hand.

Reply:
I would prefer that we not fetch packages like this.  You can get a series from a dataframe by accessing one of the columns:

```python
s = df[df.columns[0]]
```